unit_test.py

```diff
@@ -0,0 +1,1371 @@
+# This is file contains a unit test suit which completely test the backend
```

Comment:
Nix this sentence. It repeats the information found in the name of the file, and in the first import statement.